tests/test_efo.py

```diff
@@ -28,6 +28,91 @@ def testGetEFO(self):
             for path in json_response['path_codes']:
                 self.assertEqual(path[-1],id, 'disease found')
 
+    def testPostEFO(self):
+
+        diseases = ['EFO_0001365','EFO_0003966','HP_0000118','EFO_0000508','Orphanet_71862']
+
+        fields = ['label', 'code']
+        response = self._make_request('/api/latest/private/disease',
+                                      data=json.dumps(
+                                          {'diseases': diseases, 'facets': 'true', 'fields': fields}),
+                                      content_type='application/json',
+                                      method='POST',
+                                      token=self._AUTO_GET_TOKEN)
+        self.assertTrue(response.status_code == 200)
```

Comment:
the test should not only assert that the method responds, but also that the object returned makes sense. For instance, checking the presence of facets. Or a particular label for a given gene.